bundles/org.openhab.ui/web/src/assets/definitions/blockly/blocks-eventbus.js

```diff
@@ -29,9 +35,11 @@ export default function (f7, isGraalJs) {
     const itemName = javascriptGenerator.valueToCode(block, 'itemName', javascriptGenerator.ORDER_ATOMIC)
     const value = javascriptGenerator.valueToCode(block, 'value', javascriptGenerator.ORDER_ATOMIC)
     if (isGraalJs) {
-      return `items.getItem(${itemName}).${eventType}(${value});\n`
+      addItemObject(isGraalJs)
+      return `_itemObject(${itemName}).${eventType}(${value});\n`
```

Comment:
In which case do you pass an Item object to the send command/post update block?

Reply:
The problem is that the naming of the blocks, which we cannot change, is kind of confusing, so it isn't necessarily clear whether you should apply the item-block, which is basically only a name or the while getItem-Block, which returns the whole item. This approach allows to use either one. This has often been an issue in forums, so by this approach the blocks that take an item become more forgiving for beginners.

 Also, there would be the usecase to get all members of an item and then iterate over them and send a particular command to all of them.